packages/core/src/scss/components/checkout/checklist/_checklist.scss

```diff
@@ -33,6 +33,16 @@
     background-color: container("fill", "dark");
 }
 
+.custom-checklist-item {
+    margin: 0;
+    overflow: hidden;
```

Comment:
I noticed that this class is being used as `form-checklist-item optimizedCheckout-form-checklist-item custom-checklist-item` in checkout-js.

I also saw that `overflow: hidden;` is defined at L20. Is this redundant?

Could you explain how `margin: 0;` won’t cause any unwanted side effects for the other payment methods?

Reply:
Hi @bc-peng 
You are right, we don't need `overflow: hidden;` - removed.

`margin: 0;` won't case any effects to  the other payment methods because for now we are using this style with margin 0 for our list items. So this option removed as well, because we already have it.